index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+var semver = require('semver');
+var execa = require('execa');
+var del = require('del');
+
+var exec = function (cmd, args) {
+	const result = execa.sync(cmd, args);
+
+	if (result.stderr !== '') {
```

Comment:
It's used in the `if` statements.

```js
if (exec('git', ['status', '--porcelain']) !== '') {
```

So just do something like this and print all the output?
```js
const exec = (cmd, args) => {
	const {stdout, stderr} = execa.sync(cmd, args);
	console.log(stdout);
	console.log(stderr);        

	return stdout;
};
```